packages/sdk-rtl/src/paging.ts

```diff
@@ -363,7 +363,7 @@ export class Paging<TSuccess extends ILength, TError>
   }
 
   parse(raw: IRawResponse): IPager<TSuccess, TError> {
-    const req = new URL(raw.url)
+    const req = new URL(raw.url, 'https://localhost')
```

Comment:
hm, do we have access to the settings `base_url` here?

Reply:
Exactly my thought, as well. Sorry, I should have responded to the PR rather than via chat.